mkdocs.yml

```diff
@@ -27,6 +27,7 @@ nav:
             - Environmental Art: tutorials/art/environmental_art.md
             - Modeling Basics: tutorials/art/modeling_basics.md
             - Audio: tutorials/art/audio_reference.md
+            - Object Generator: tutorials/art/object_generator.md
```

Comment:
need to add this in prod as well, and the file name is actually objectgenerator.md without the underscore, but I would prefer having the underscore I think